mscore/inspector/inspectorBend.h

```diff
@@ -34,7 +34,19 @@ class InspectorBend : public InspectorElementBase {
    public:
       InspectorBend(QWidget* parent);
       const QList<PitchValue>& points() const;
+      virtual void setElement() override;
       };
 
+static const QList<PitchValue> BEND 
+   = { PitchValue(0, 0),   PitchValue(15, 100), PitchValue(60, 100) };
+static const QList<PitchValue> BEND_RELEASE
+   = { PitchValue(0, 0),   PitchValue(10, 100), PitchValue(20, 100), PitchValue(30, 0), PitchValue(60, 0) };
+static const QList<PitchValue> BEND_RELEASE_BEND
+   = { PitchValue(0, 0),   PitchValue(10, 100), PitchValue(20, 100), PitchValue(30, 0), PitchValue(40, 0), PitchValue(50, 100), PitchValue(60, 100) };
+static const QList<PitchValue> PREBEND
+   = { PitchValue(0, 100), PitchValue(60, 100) };
+static const QList<PitchValue> PREBEND_RELEASE
+   = { PitchValue(0, 100), PitchValue(15, 100), PitchValue(30, 0),   PitchValue(60, 0) };
```

Comment:
If these variables are only needed for `InspectorBend` it would be better to declare them in `inspectorBend.cpp` or make them member variables within `InspectorBend` class.

Reply:
Making them member variables makes more sense, but if I remember correctly, it's more common in MuseScore code to declare such things in the cpp file. So I kind of cannot determine which should I follow...

Reply:
I would say that any of these options would be good, I don't see strong reasons to prefer one or another option and there seems to be no real convention for that within MuseScore code. The important thing to me here is to remove these variables from the relatively global `Ms` namespace scope.

Reply:
I don't actually see why they should be removed from `Ms` namespace. The `static const` variable here is in `Ms`:

https://github.com/musescore/MuseScore/blob/master/libmscore/layoutbreak.cpp#L18-L30

Reply:
Maybe I explained my point a bit incorrectly. That variable is indeed in `Ms` namespace but it is declared `static` and therefore will be available only within one translation unit (the one for `layoutbreak.cpp`), so it will not be visible in other `.cpp` files. The variables in `inspectorBend.h` are also `static` but they are declared in a header file so every file that includes (directly or via other headers) `inspectorBend.h` will have these `BEND` variables in `Ms` namespace. This is exactly what I would like to avoid, and either moving them to `.cpp` file or making them static class members would resolve this issue.

Reply:
Got it. Will move them to `.cpp` file.